docs/semgrep-sc/sc-getting-started.md

```diff
@@ -62,6 +67,7 @@ Read [Step 3](#step-3-enabling-semgrep-supply-chain-scans-from-within-semgrep-ap
 
 <PlatformSigninGithub />
 
+
 See [Permissions in GitHub](/docs/semgrep-app/getting-started-with-semgrep-app/#permissions-for-github) to learn more about how Semgrep features use requested permissions in GitHub.
```

Comment:
```suggestion
See [Permissions in GitHub](/semgrep-app/getting-started-with-semgrep-app/#permissions-for-github) to learn more about how Semgrep features use requested permissions in GitHub.
```

Reply:
Hope we can make vale.sh lint the docs one day so we don't have to do this...